x-pack/legacy/plugins/siem/server/lib/detection_engine/signals/build_rule.test.ts

```diff
@@ -192,8 +199,84 @@ describe('buildRule', () => {
       updatedBy: 'elastic',
       interval: 'some interval',
       tags: ['some fake tag 1', 'some fake tag 2'],
+      throttle: null,
     });
     const expected: Partial<OutputRuleAlertRest> = {
+      actions: [],
+      created_by: 'elastic',
+      description: 'Detecting root and admin users',
+      enabled: true,
+      false_positives: [],
+      from: 'now-6m',
+      id: '04128c15-0d1b-4716-a4c5-46997ac7f3bd',
+      immutable: false,
+      index: ['auditbeat-*', 'filebeat-*', 'packetbeat-*', 'winlogbeat-*'],
+      interval: 'some interval',
+      language: 'kuery',
+      max_signals: 10000,
+      name: 'some-name',
+      note: '',
+      output_index: '.siem-signals',
+      query: 'user.name: root or user.name: admin',
+      references: ['http://google.com'],
+      risk_score: 50,
+      rule_id: 'rule-1',
+      severity: 'high',
+      tags: ['some fake tag 1', 'some fake tag 2'],
+      to: 'now',
+      type: 'query',
+      updated_by: 'elastic',
+      version: 1,
+      updated_at: rule.updated_at,
+      created_at: rule.created_at,
+      lists: [
+        {
+          field: 'source.ip',
+          boolean_operator: 'and',
+          values: [
+            {
+              name: '127.0.0.1',
+              type: 'value',
+            },
+          ],
+        },
+        {
+          field: 'host.name',
+          boolean_operator: 'and not',
+          values: [
+            {
+              name: 'rock01',
+              type: 'value',
+            },
+            {
+              name: 'mothra',
+              type: 'value',
+            },
+          ],
+        },
+      ],
+    };
+    expect(rule).toEqual(expected);
+  });
+
+  test('it omits a null value such as if throttle is undefined if is present', () => {
```

Comment:
Test description is unclear; what is this testing that the previous test is not?

Reply:
added `"` around the keys we're checking, did it help?